docs/README-dev.md

```diff
@@ -217,22 +221,30 @@ Next, bring in the Narrative and Bootstrap repos (as yourself):
     git clone http://github.com/kbase/bootstrap
     git clone http://github.com/kbase/ui-common
 
+# note that as of 2/10/15, narrative-test.kbase.us is using the
+# develop branch of both narrative and ui-common
+
 # Set up the Nginx environment
 
 This uses Nginx with the embedded Lua module from the nginx-extras package, described here: <http://wiki.nginx.org/Install>.
 
-
 ## Install Nginx
 
     sudo su - root
     # these next two should add the nginx repo properly
-    apt-get install-y  python-software-properties
+    apt-get install -y  python-software-properties
     add-apt-repository ppa:nginx/stable
     apt-get update
     apt-get install -y nginx-extras
     # verify this worked before going on
     exit
 
+## Install the right version of Nodejs (including npm)
+
+   sudo add-apt-repository ppa:chris-lea/node.js
```

Comment:
There's an official ppa, now - see here: https://github.com/joyent/node/wiki/installing-node.js-via-package-manager

Also, that should only be for Ubuntu 12.04 builds. I thought I updated the instructions to grab a 14.04 Ubuntu image, though I could be wrong. Either way, 14.04 would be a better target.